server/src/test/resources/permissions.json

```diff
@@ -13,5 +13,10 @@
       "hono-client": [ "READ", "WRITE" ],
       "rest-adapter": [ "READ", "WRITE" ],
       "mqtt-adapter": [ "READ" ]
+  },
+  "credentials/DEFAULT_TENANT": {
+    "hono-client": [ "READ", "WRITE" ],
+    "rest-adapter": [ "READ", "WRITE" ],
```

Comment:
Will change (and especially provide the permissions file for the Hono app itself, this one was only for the tests so far)